src/test/resources/org/jabref/logic/importer/fileformat/RisImporterTest1.ris

```diff
@@ -25,10 +25,9 @@ PY  - 2001/00/00/
 PY  - ///
 PY  - 2001///
 Y1  - 2001/09/01/
-AU  - Cocke 
-A1  - Harrison, Warren 
-Levenshtein
-        Morris
+AU  - Cocke
+A1  - Harrison, Warren
+Levenshtein Morris
```

Comment:
Please do not modify test files - they come from other tools - and should still work

Add unig tests closer to your code to cover this case